### ISSUE TITLE
[Egg Request]: Stay In Tarkov

### ISSUE THREAD

**Issue by cocorico8** [parkervcp/eggs#2580]
### Does this expand an already existing service

No

### Link to a game or software

https://github.com/stayintarkov/SIT.Aki-Server-Mod

### Links for the download

https://github.com/stayintarkov/SIT.Aki-Server-Mod/archive/refs/heads/master.zip

### Links for the install docs

https://github.com/stayintarkov/SIT.Aki-Server-Mod/blob/master/README.md

**Comment by QuintenQVD0** [comment]
> `This repository has been archived by the owner on Nov 18, 2023. It is now read-only.`

Where do you see that?
![afbeelding](https://github.com/parkervcp/eggs/assets/67589015/e4a17071-5a6a-4af1-a135-3d3c055e93ee)

**Comment by QuintenQVD0** [comment]
We will not create eggs for archived projects.

**Comment by cocorico8** [comment]
hum i don't understand

**Comment by cocorico8** [comment]
ok i understand so they had little problem with the game owner and they change the repositery to an organisation
everything is here [https://github.com/stayintarkov](url) and the main project is here [https://github.com/stayintarkov/StayInTarkov.Client](url)

**Comment by cocorico8** [comment]
they forgot to modify the readme

**Comment by QuintenQVD0** [comment]
I marked this as to litte / good documentation

**Comment by cocorico8** [comment]
can you rephrase your sentence ?

> TO be honnest there documentation for this is trash and all over the place on how to set this up

**Comment by gOOvER** [comment]
> can you rephrase your sentence ?
> 
> > TO be honnest there documentation for this is trash and all over the place on how to set this up

the docs are trash :D Its not really clear, how to install what :D

**Comment by cocorico8** [comment]
what do you need to have to make an egg properly i can maybe help you understand the doc

**Comment by cocorico8** [comment]
can i make a explanation right here ?

**Comment by cocorico8** [comment]
because i don't really understand how

**Comment by cocorico8** [comment]
what part

**Comment by gOOvER** [comment]
> what part

all? updated links?

**Comment by cocorico8** [comment]
of course, i will look into pterodactyl docs see if i can understand the egg.

**Comment by QuintenQVD0** [comment]
> of course, i will look into pterodactyl docs see if i can understand the egg.

If i can give you a hand look at the github repo. There is a pr to update the eggs docs as there a litte old. 

What I meant earlyer, I have made some eggs or fixt a some but to be honnest if people request an egg and we need to literly look left and right for documentation that are the egg request that stay open. If someone realy needs an egg for this then they will dedicated time to it. As making an egg already takes a fare amount of time and then even extra time to look for docs that is a step to far for many. But if you want to do it go ahead have fun and if we see it in pr I br happy to revieuw it.

**Comment by QuintenQVD0** [comment]
> can you hand the link ?

Pr: https://github.com/pterodactyl/documentation/pull/556

My own remake of the docs: https://quintenqvd0.github.io/pterodactyl-documentation/docs/eggs/egg/

**Comment by gOOvER** [comment]
When i read correct, you need to upload the full game too

**Comment by cocorico8** [comment]
they talk about having the client but it is just for the mod client side, the egg don't need that

**Comment by gOOvER** [comment]
> they talk about having the client but it is just for the mod client side, the egg don't need that

pls show me that part. In my tests i need all the time the full game uloaded

**Comment by cocorico8** [comment]
wdym

**Comment by gOOvER** [comment]
> wdym

link me the guide, where i see, that ONLY the server is needed, not the full game

**Comment by cocorico8** [comment]
[https://discord.com/channels/1175114933713776690/1178076854926717098/1178076854926717098](url) here on there discord [https://discord.gg/7gYqA8wgst](url)

**Comment by gOOvER** [comment]
> [https://discord.com/channels/1175114933713776690/1178076854926717098/1178076854926717098](url) here on there discord [https://discord.gg/7gYqA8wgst](url)

sry, i will not join a discord for Installguide.

In my Tests in everytime nneds the full game and i will not upload 20gb

**Comment by gOOvER** [comment]
> if you want i have an archive with the game the launcher and the server set up

i want a guide, not an archive :)

**Comment by cocorico8** [comment]
> > [https://discord.com/channels/1175114933713776690/1178076854926717098/1178076854926717098](url) here on there discord [https://discord.gg/7gYqA8wgst](url)
> 
> sry, i will not join a discord for Installguide.
> 
> In my Tests i everytime needs the full game and i will not upload 20gb

it's not a steam game you just need the spt server, add the mod to it and open the port 6969 and 6970

**Comment by ImKringle** [comment]
Odds are this runs the same as Hogwarp did - Devs putting files and all docs *if any* into a discord instead of a site that is well made



> > if you want i have an archive with the game the launcher and the server set up
> 
> i want a guide, not an archive :)

StayInTarkov Coop Mod Installation Guide for Host (Per their Discord)
Downloads:
[SPT AKI 3.8](https://pixeldrain.com/u/QdkVUzeu) or [SPT AKI 3.7.3](https://dev.sp-tarkov.com/attachments/a3ddb3cd-06ee-423f-babf-a06e1f101a9f)
[SIT.Aki-Server-Mod](https://github.com/stayintarkov/SIT.Aki-Server-Mod/archive/refs/heads/master.zip)
[SIT.Manager](https://github.com/stayintarkov/SIT.Manager/releases/latest/download/SIT.Manager.zip) - [ Requires .Net8](https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/runtime-desktop-8.0.0-windows-x64-installer)

Installation Steps:

Make sure Escape from Tarkov is up to date.
Download the three ZIP files from above.
Create three folders named: game, server, launcher.
Copy your live Escape from Tarkov files into the game folder.
[DOWNGRADE PATCHER 0.13.9.1.27622 to 0.13.9.1.27050](https://pixeldrain.com/u/MFJcZDuy)
Extract it in the same folder where you copy the live EFT and run the patcher.exe
Extract the contents of Aki3.8or3.7.3 into the server folder.
Extract the contents of SIT.Server-Aki-Mod into server\user\mods\SITCoop. If there's an existing folder, delete it first. Run Aki.Server.exe once to generate config files. Wait until it says "happy playing," then close it.
Extract the contents of SIT.Manager into the launcher folder.
Configure IP Settings:
Modify server\Aki_Data\Server\configs\http.conf with your local or Hamachi/Radmin IP address if you're using them.
Modify server\user\mods\SITCoop\config\coopConfig.json to your external IP address or Hamachi/Radmin IP.
Port Forwarding (If Not Using Hamachi/Radmin):
If not using Hamachi or Radmin, port forward 6969 and 6970. Also, disable Windows Firewall.
Start the server\aki.server.exe.
Start the launcher (SIT.Manager), specify the game folder path in the settings tab, ensuring it indicates EFT version 0.13.9.xxx.
Install SIT from the tools tab.
Enter the IP and connect.

These steps definitely need the base game files to even work - this is likely gonna be a open egg req for a while. Weird they cant just make the server setup a basic shell; its all nodejs based :weird:

**Comment by ImKringle** [comment]
Actually, this would be easier; seems doable - assuming this runs on Wine (as nothing here supports linux. Go figure.)
Uploading and Installing SIT-Server:
Download [Aki-Server](https://pixeldrain.com/u/QdkVUzeu) and [SIT-Server-Mod](https://github.com/stayintarkov/SIT.Aki-Server-Mod/archive/refs/heads/master.zip) or Archive your SIT-server on your PC.
If you already have a server set-up on your PC upload the server archive on (e.g., [EasyUpload](https://easyupload.io/)).
Copy the uploaded file link.
Access the VPS browser (comes with Chrome) and download 7zip or [WinRAR](https://www.win-rar.com/fileadmin/winrar-versions/winrar/winrar-x64-624.exe).
Download the SIT-server archive using the link.
Extract the archive on the VPS desktop or your preferred location.

Configuring SIT-Server:
Modify server\Aki_Data\Server\configs\http.conf to use your VPS IP.
Update server\user\mods\SITCoop\config\coopConfig.json with your VPS IP.

Adjusting Firewall Settings:
Open Windows Security Firewall & network protection. (Click on Start, then type "Windows Security.")
Turn off the Public network firewall.

Restarting VPS and Launching Aki-Server:
Restart your VPS and wait for 30 seconds to 1 minute.
Reconnect to your VPS.
Launch Aki-Server.exe

**Comment by ovingiv** [comment]
Someone did make a docker for SIT. https://hub.docker.com/r/stayintarkov/stayintarkov

**Comment by papertots** [comment]
> Someone did make a docker for SIT. https://hub.docker.com/r/stayintarkov/stayintarkov

I made an egg based on the dockerfile from there, [here's the pull request](../pull/2753). I can connect to it and enter a raid with no issues, but I can't really test any further since I've never played tarkov before. Would be happy if anyone else with a copy of the game could test it, thanks!

**Comment by ovingiv** [comment]
It's working for me. Added mods without an issue and it loaded up.
